client/components/signup-form/index.jsx

```diff
@@ -10,6 +10,7 @@ import keys from 'lodash/keys';
 import debugModule from 'debug';
 import classNames from 'classnames';
 import i18n from 'i18n-calypso';
+import store from 'store';
```

Comment:
The rebase was just to rebase the code on #6124 and didn't include the new code. The need for `localStorage` has been removed in 9cdb59e, where `suggestedUsername` has been moved as a reducer in Redux.